lib/public/components/Log/linkButton.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+import { iconLinkIntact } from '/js/src/icons.js';
+import { clipboardTarget } from '/components/common/clipboardTarget.js';
+
+/**
+ * Copies the url of the log to the clipboard
+ * @param {Log} log the log object that this button corresponds to
+ * @return {Component} The link button.
+ */
+export const linkButton = (log) => {
```

Comment:
As I said, you should NEVER do supposition on the context in the components. 
When you will use this component to be able to copy log link from runs details page, log tab, will this still work?

Also, when I will want to create a run link button, can I still name it `linkButton`? It will become hard to chose when using autocomplete, and I will have to look at the import declaration when reading code using this!